testing/test_debugging.py

```diff
@@ -1373,6 +1373,7 @@ def do_quit(self, *args):
     result.stdout.fnmatch_lines(["*runcall_called*", "* 1 passed in *"])
 
 
+@pytest.mark.xfail(sys.version_info >= (3, 14), reason="I don't know why this fails")
```

Comment:
pdb/bdb changed a lot in 3.14 not sure what this is

Reply:
This change will need discussion on how to resolve: https://github.com/pytest-dev/pytest/issues/13308#issuecomment-2911867744